src/services/functionAppService.ts

```diff
@@ -257,7 +268,16 @@ export class FunctionAppService extends BaseService {
   public async updateFunctionAppSetting(functionApp: Site, setting: string, value: string) {
     const { properties } = await this.webClient.webApps.listApplicationSettings(this.resourceGroup, functionApp.name);
     properties[setting] = value;
-    await this.webClient.webApps.updateApplicationSettings(this.resourceGroup, functionApp.name, properties);
```

Comment:
This SDK was failing even though the params were correct. Would like to use an SDK call instead of making our own rest API request, but needed to do so in order to unblock

Reply:
Do we know why it was failing or what is the difference between the SDK call vs. the manual REST API call?